cmd/clusterctl/internal/test/fake_variable_client.go

```diff
@@ -17,9 +17,18 @@ limitations under the License.
 package test
 
 import (
+	"fmt"
+
 	"github.com/pkg/errors"
 )
 
+var (
+	// PrivateGithubDomain is the private github enterprise domain used int tests.
```

Comment:
What does int tests mean here? If it's an abbreviation, please expand it so it's better documented.